target/linux/pistachio/base-files/etc/init.d/bt-csr

```diff
@@ -139,22 +139,16 @@ EOF
 		}
 
 	fi
-
-# Configure HCI and get some version info
-	BT_ADDR_MASK=10
-	hciconfig hci0 reset &&\
-	hciconfig hci0 up &&\
-	hciconfig hci0 piscan &&\
-	hciconfig hci0 version &&\
-	BT_ADDR=$(echo $MAC | cut -c $BT_ADDR_MASK-) &&\
-	BT_NAME="Creator-Ci40["$BT_ADDR"]" &&\
-	hciconfig hci0 name $BT_NAME &&\
-	hcitool dev || (echo  "Failed to configure" && exit 1)
-
 }
 
 start() {
 	echo  "Starting Bluetooth..."
+	lsmod | grep hci_uart
+	while [ $? -ne 0 ]
+	do
+		lsmod | grep hci_uart
```

Comment:
I don't think redirect output is necessary if you use ```-q``` option of grep. Can you try that @tjobanpu  ?